gateway-js/src/config.ts

```diff
@@ -145,17 +138,12 @@ export interface RemoteGatewayConfig extends GatewayConfigBase {
 // TODO(trevor:cloudconfig): This type goes away
 export interface LegacyManagedGatewayConfig extends GatewayConfigBase {
   federationVersion?: number;
+  schemaConfigDeliveryEndpoint: null;
 }
 
 // TODO(trevor:cloudconfig): This type becomes the only managed config
 export interface PrecomposedManagedGatewayConfig extends GatewayConfigBase {
-  /**
-   * @deprecated This configuration option shouldn't be used unless by
-   *             recommendation from Apollo staff. This behavior will be
-   *             defaulted in a future release and this option will strictly be
-   *             used as an override.
-   */
-  experimental_schemaConfigDeliveryEndpoint: string;
+  schemaConfigDeliveryEndpoint?: string;
```

Comment:
Do we still want a comment saying not to set this unless told by staff? Or to say basically "set to null to roll back to the pre-0.whatever state, don't set to a string unless told to by staff"?

Reply:
9f9a2b06b8bec3e5f2cbe2038c96036c5b0ed22b